### ISSUE TITLE
New Export Audio dialog: missing accessibility names

### ISSUE THREAD

**Issue by DavidBailes** [audacity/audacity#4955]
### Steps to reproduce

Using this commit: https://github.com/audacity/audacity/commit/d3d98a434a0cf9511526842b70799234c0cdbba4, which hasn't yet been merged into master, and using a screen reader on Windows:
1. Open the Export Audio dialog
2. Set the format to Custom FFmpeg Export
3. Tab to the Current format and Current Codec text boxes.

### Expected behavior

The names of the text boxes, Current Format and Current Codec, are read by the screen reader

### Actual behavior

The names of the text boxes are not read.

### Audacity Version

other

### Operating system

Windows

### Additional context

This bug is in the commit: https://github.com/audacity/audacity/commit/d3d98a434a0cf9511526842b70799234c0cdbba4,
which hasn't yet been merged into master.

**Comment by LWinterberg** [comment]
This bug also is present in 3.3.3

**Comment by chinakov** [comment]
@DavidBailes  - thank you, the issue has been fixed in the latest branch build. Please merge.

**Comment by DavidBailes** [comment]
> @DavidBailes - thank you, the issue has been fixed in the latest branch build. Please merge.

I suspect that I don't have to permissions to merge, and someone else will have to do this.

**Comment by vsverchinsky** [comment]
@chinakov sorry, missed that message, done now
